src/test/java/com/ss/greencity/util/WaitsSwitcher.java

```diff
@@ -0,0 +1,91 @@
+package com.ss.greencity.util;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.concurrent.TimeUnit;
+import java.util.function.Function;
+
+/**
+ * A class intended to help easily switching between implicit and explicit waits
+ */
+public final class WaitsSwitcher {
+    private WebDriver driver;
+    private long defaultImplicitWait;
+    private long defaultExplicitWait;
+
+    public WaitsSwitcher(WebDriver driver, long defaultImplicitWait, long defaultExplicitWait) {
+        this.driver = driver;
+        this.defaultImplicitWait = defaultImplicitWait;
+        this.defaultExplicitWait = defaultExplicitWait;
+    }
+    public WaitsSwitcher(WebDriver driver, long defaultWait) {
+        this.driver = driver;
+        defaultImplicitWait = defaultWait;
+        defaultExplicitWait = defaultWait;
+    }
+    public WaitsSwitcher(WebDriver driver) {
+        this.driver = driver;
+        defaultImplicitWait = 1;
+        defaultExplicitWait = 5;
+    }
+
+    /**
+     * A wrap for Thread.sleep
+     * @param seconds - time in seconds (could be decimal)
+     */
+    public static void sleep(double seconds) {
+        long millis = secondsToMillis(seconds);
+        try {
+            Thread.sleep(millis);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * A wrap for driver.manage().timeouts().implicitlyWait
+     * @param seconds - time in seconds
+     */
+    public void setImplicitWait(long seconds) {
+        if (seconds > 0) {
+            defaultImplicitWait = seconds;
+        }
+        driver.manage().timeouts().implicitlyWait(seconds, TimeUnit.SECONDS);
+    }
+    /**
+     * A wrap for driver.manage().timeouts().implicitlyWait
+     */
+    public void setImplicitWait() {
+        setImplicitWait(defaultImplicitWait);
+    }
+
+    /**
+     * Nullifies implicit wait, explicitly waits for a web element or an event, and then restores the implicit wait.
+     * Also returns the object waited for
+     * @param seconds - time in seconds
+     * @param expectedCondition - boolean function, something like ExpectedConditions.presenceOfElementLocated()
+     */
+    public <V> V setExplicitWait(long seconds, Function<? super WebDriver, V> expectedCondition) {
```

Comment:
ATTENTION! This is what you need to use instead of Thread.sleep(). 
This waits explicitly, but then returns implicit wait, which is nice.
As a Function parameter, you should use something from the class ExpectedConditions